src/css/imports/jwplayerelement.less

```diff
@@ -148,7 +148,12 @@
 
 // These items can intercept pointer-events
 .jw-overlays > div,
-.jw-media, .jw-controlbar, .jw-dock, .jw-logo, .jw-skip, .jw-display-icon-container,
+.jw-media,
+.jw-controlbar,
+.jw-dock > div,
```

Comment:
These types are queries can be very inefficient (from every div check if the parent has a .jw-dock parent). 

Let's make it the elements with the class we know we add directly to these containers:

```css
.jw-overlays .jw-plugin,
.jw-dock .jw-dock-button,
```